infrastructure/main.tf

```diff
@@ -32,19 +37,12 @@ locals {
 
 data "azurerm_key_vault" "ccd_shared_key_vault" {
   name = "${local.vaultName}"
-  resource_group_name = "${local.vaultName}"
-}
-
-data "vault_generic_secret" "definition_store_item_key" {
-  path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/ccd-definition"
+  resource_group_name = "${local.sharedResourceGroup}"
 }
 
-data "vault_generic_secret" "gateway_idam_key" {
-  path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/ccd-gw"
-}
-
-data "vault_generic_secret" "gateway_oauth2_client_secret" {
-  path = "secret/${var.vault_section}/ccidam/idam-api/oauth2/client-secrets/ccd-gateway"
+data "azurerm_key_vault_secret" "definition_store_item_key" {
```

Comment:
It says `item` instead of `idam`, and actually, if it's s2s it's not idam anymore; so something closer to the actual secret name would be better I reckon.